p2p_wallet/Services/GlobalAppState.swift

```diff
@@ -18,8 +18,9 @@ class GlobalAppState: ObservableObject {
     
     // Debug features
     @Published var forcedWalletAddress: String = ""
-    @Published var forcedFeeRelayerEndpoint: String = "" {
+    @Published var forcedFeeRelayerEndpoint: String? = Defaults.forcedFeeRelayerEndpoint {
```

Comment:
Default value must be nil

Reply:
It is nil if it is not set

Reply:
Testers asked to store the choice if it is made via debug menu